bootcampspringboot/src/main/java/com/max/bootcampspringboot/api/EmployeeController.java

```diff
@@ -0,0 +1,44 @@
+package com.max.bootcampspringboot.api;
+
+import com.max.bootcampspringboot.api.mapper.ApiEmployeeMapper;
+import com.max.bootcampspringboot.api.model.ApiEmployee;
+import com.max.bootcampspringboot.service.EmployeeService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/employees")
+public class EmployeeController {
+    private final EmployeeService employeeService;
+    public EmployeeController(EmployeeService employeeService) {
+        this.employeeService = employeeService;
+    }
+
+    @GetMapping("/{id}")
+    public ApiEmployee getEmployee(@PathVariable int id){
+        return ApiEmployeeMapper.toApiEmployee(employeeService.getEmployee(id));
+    }
+
+    @GetMapping
+    List<ApiEmployee> getAllEmployees(){
+        return ApiEmployeeMapper.toApiEmployee(employeeService.getAllEmployees());
+    }
+
+    @PostMapping
+    public ApiEmployee addEmployee(@RequestBody ApiEmployee employee){
+        return ApiEmployeeMapper.toApiEmployee(employeeService.addEmployee(ApiEmployeeMapper.toServiceEmployee(employee)));
+    }
+
+    @PutMapping
```

Comment:
Es wäre besser die Id mit in den Pfad aufzunehmen und diese dann ins Objekt zu legen oder die Ids zu überprüfen, ob diese gleiche sind. Ist ein Sicherheitsfaktor.